client/src/components/Modals/ReviewUser/index.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { ReactElement, FC, useState } from 'react';
+import {
+  Button,
+  Grid,
+  Modal,
+  ModalProps,
+  Icon,
+  Form,
+  Input,
+} from 'semantic-ui-react';
+import { IUser, ITeam } from 'ssw-common';
+
+import { updateUser, updateOnboardingStatus } from '../../../api';
+import { useInterests, useTeams } from '../../../contexts';
+import { UserPicture, FieldTag } from '../..';
+import './styles.scss';
+import { getUserFullName, titleCase } from '../../../utils/helpers';
+
+interface ReviewUserProps extends ModalProps {
+  user: IUser;
+}
+
+const ReviewUserModal: FC<ReviewUserProps> = ({ user }): ReactElement => {
+  const [isOpen, setIsOpen] = useState(true);
+  const [formValue, setFormValue] = useState('');
+
+  const { getTeamFromId } = useTeams();
+  const { getInterestById } = useInterests();
+
+  const approveUser = (u: IUser): void => {
+    updateOnboardingStatus(u._id, 'ONBOARDED');
+    setIsOpen(false);
```

Comment:
What if the endpoint fails? You are going to close the modal when the user wasn't updated